node/src/chain_spec/mangata_rococo.rs

```diff
@@ -348,23 +335,73 @@ pub fn mangata_rococo_local_config() -> ChainSpec {
 					),
 				],
 				vec![
-					(ROC_TOKEN_ID, None),
-					(5, None),
+					(
+						0,
+						AssetMetadataOf {
+							decimals: 18,
+							name: b"Mangata".to_vec(),
+							symbol: b"MGA".to_vec(),
+							additional: EmptyCustomMetadata,
+							existential_deposit: fee::MGX_BASE_EXISTENTIAL_DEPOSIT,
+							location: None,
+						},
+					),
+					(
+						1,
+						AssetMetadataOf {
+							decimals: 18,
+							name: b"Ether".to_vec(),
+							symbol: b"ETH".to_vec(),
+							additional: EmptyCustomMetadata,
+							existential_deposit: 0,
+							location: None,
+						},
+					),
+					(
+						ROC_TOKEN_ID,
+						AssetMetadataOf {
+							decimals: 12,
+							name: b"Rococo Native".to_vec(),
+							symbol: b"ROC".to_vec(),
+							additional: EmptyCustomMetadata,
+							existential_deposit: 1_000_000_000,
```

Comment:
yes we don't implement existential deposits,
here the `existential_deposit` serves only the purpose of computing the xcm fee rate
eg. we have EDs for 10MGR & 0.01KAR -> we can then compute the ~rate_per_second for KAR token
or any other registered token

the `existential_deposit` is a bit misleading, but wanted to keep the naming standard
the other option might be to use the `additional` field to store the fee directly instead of computation

Reply:
As discussed... Let's use the additional field, Option<u128> representing tokens_per_second

Reply:
y, done